providers/default.rb

```diff
@@ -224,11 +231,41 @@ def move_opt_chef(src, dest)
   end
 end
 
+def prepare_windows
```

Comment:
Not sure you need all the guards on platform family here.  Either once at the start of the function or maybe not at all since it'll only be called if the platform is windows (as defined below).

Reply:
You are right. Actually, its only being used on windows hosts in the first place, so every guard here is redundant.